### PR TITLE
Doc: connect, disconnect, connections CLI ref

### PR DESCRIPTION
**Pull request by akcano**

### CODE REVIEW

File: cmd/workshop/connect.go [hunk 1]
```diff
@@ -14,16 +14,37 @@ type CmdConnect struct {
...
+This command connects a plug to a target slot
+that is specified as the second argument or deduced from the context.
+
+- If the second argument is omitted entirely, the target is assumed to be
+  <WORKSHOP>/agent:<PLUG>; <WORKSHOP> and <PLUG> come from the first argument
+
+- If the second argument only names the slot itself, the target is
+  <WORKSHOP>/agent:<SLOT>; <WORKSHOP> comes from the first argument
+
+- If the second argument only names the workshop and SDK, the target is
```
Comment (dmitry-lyfar) [inline cmd/workshop/connect.go:30]: In the third case, the target slot must be the only one compatible with the plug. If there are multiple slots with <INTERFACE>, connect will fail.

File: cmd/workshop/connections.go [hunk 2]
```diff
@@ -17,14 +17,24 @@ type CmdConnections struct {
...
 	var cmd = &cobra.Command{
-		Use:     "connections [<WORKSHOP>] [--all]",
-		Args:    cobra.MaximumNArgs(1),
-		Short:   "List interface connections.",
+		Use:   "connections [<WORKSHOP>]",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "List interface connections.",
+		Long: `
+This command lists connections between interface plugs and slots
+globally or for a single workshop; each line is a connection between a plug
```
Comment (dmitry-lyfar) [inline cmd/workshop/connections.go]: In our context 'globally' means across projects (see workshop list --global which would show all the projects' workshops). Should it be for a project or for a single workshop? The connections command does not have a global option compared to list.

File: docs/reference/cli/workshop-connections.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+.. code-block:: console
+
+   $ workshop connect <WORKSHOP>/<SDK>:<PLUG> [<WORKSHOP>/<SDK>][:<SLOT>] [OPTIONS]
+
+
+Synopsis
+--------
+
+This command lists connections between interface plugs and slots
+globally or for a single workshop; each line is a connection between a plug
```
Comment (dmitry-lyfar) [inline docs/reference/cli/workshop-connections.rst]: Same note about using 'globally' here.

**Review by dmitry-lyfar — COMMENTED** [review]
Nice one. I suspect you defer references to the agent SDK's documentation until it exists (e.g. in the commands reference)?

**Comment by akcano** [comment]
> Nice one. I suspect you defer references to the agent SDK's documentation until it exists (e.g. in the commands reference)?

Yes, the agent-related stuff will be added separately.

File: cmd/workshop/connections.go (hunk 2 above)
Comment (akcano) [reply cmd/workshop/connections.go]: I suggest we follow what `snap connections` does, i.e. list all the snaps/workshops.

File: docs/reference/cli/workshop-connections.rst (hunk 3 above)
Comment (akcano) [reply docs/reference/cli/workshop-connections.rst]: See above.

**Review by dmitry-lyfar — APPROVED** [review]

**Pull request merged**